packages/plugins/typescript/graphql-request/src/visitor.ts

```diff
@@ -122,19 +124,27 @@ export class GraphQLRequestVisitor extends ClientSideBaseVisitor<
       o.operationResultType
     }>(${docArg}, variables, {...requestHeaders, ...wrappedRequestHeaders}), '${operationName}', '${operationType}');
 }`;
-        }
-        return `${operationName}(variables${optionalVariables ? '?' : ''}: ${
-          o.operationVariablesTypes
-        }, requestHeaders?: Dom.RequestInit["headers"]): Promise<${o.operationResultType}> {
+        } else if (this.config.generateIndividualOperations) {
```

Comment:
I'm not comfortable with this implementation that "hijack" the SDK object properties building.

The code would be easier to read if the operations exports were handled earlier, for example, at line `111`.
Also, is there any motivation for dropping the support of wrappers for this new option?